lib/wallet/api/http/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -30,11 +30,6 @@ import Cardano.Address.Script
     ( Cosigner (..) )
 import Cardano.Ledger.Alonzo.TxInfo
     ( TranslationError (..) )
-import Cardano.Tx.Balance.Internal.CoinSelection
-    ( SelectionBalanceError (..)
-    , UnableToConstructChangeError (..)
-    , WalletSelectionContext
-    )
```

Comment:
We no longer rely on internal coin selection modules in the wallet HTTP error API. 🍾